src/components/dashboard/sidebar/LearnerCreditSummaryCard.jsx

```diff
@@ -46,100 +44,124 @@ const badgeStatusMessages = defineMessages({
   expirationDate,
   statusMetadata,
   assignmentOnlyLearner,
 }) => {
-  const { status, badgeVariant } = statusMetadata;
-  const { data: enterpriseCustomer } = useEnterpriseCustomer();
   const intl = useIntl();
+  const { status, badgeVariant } = statusMetadata;
+  const { data: { disableExpiryMessagingForLearnerCredit } } = useEnterpriseCustomer();
+  const isBudgetExpired = dayjs(expirationDate).isBefore(dayjs()) && status === BUDGET_STATUSES.expired;
 
-  const cardBadge = useMemo(() => conditionallyRenderCardBadge({
-    disableExpiryMessagingForLearnerCredit: enterpriseCustomer.disableExpiryMessagingForLearnerCredit,
+  const cardBadge = getStatusBadge({
+    disableExpiryMessagingForLearnerCredit,
     status,
     badgeVariant,
     intl,
-  }), [badgeVariant, enterpriseCustomer.disableExpiryMessagingForLearnerCredit, intl, status]);
+  });
+
+  // Validates that the flag to disable expiry messaging is enabled, whether the learner credit expiration is
+  // truly expired by a date calculation and the status is set to expired
+  if (disableExpiryMessagingForLearnerCredit && isBudgetExpired) {
+    return null;
+  }
 
   return (
-    <SidebarCard
-      title={
-        (
-          <div className="d-flex align-items-center justify-content-between">
-            <h3 className="m-0">
+    <>
+      <Card.Header
+        title={
+          (
+            <Stack direction="horizontal" className="align-items-center justify-content-between">
+              <h3 className="m-0">
+                <FormattedMessage
+                  id="enterprise.dashboard.sidebar.learner.credit.card.title"
+                  defaultMessage="Learner Credit"
+                  description="Title for the learner credit summary card on the enterprise dashboard sidebar."
+                />
+              </h3>
+              {cardBadge}
+            </Stack>
+          )
+        }
+      />
+      <Card.Section>
+        <p data-testid="learner-credit-summary-text">
+          {assignmentOnlyLearner ? (
+            <FormattedMessage
+              id="enterprise.dashboard.sidebar.learner.credit.card.assignment.only.description"
+              defaultMessage="Your organization will assign courses to learners. Please contact your administrator if you are interested in taking a course."
+              description="Description for the learner credit summary card on the enterprise dashboard sidebar when learner has assignment."
+            />
+          ) : (
+            <FormattedMessage
+              id="enterprise.dashboard.sidebar.learner.credit.card.description"
+              defaultMessage="Apply your organization's learner credit balance to enroll into courses with no out of pocket cost."
+              description="Description for the learner credit summary card on the enterprise dashboard sidebar when learner has no assignment."
+            />
+          )}
+        </p>
+
+        {(expirationDate && !disableExpiryMessagingForLearnerCredit) && (
```

Comment:
Is this redundant if we're already returning null at the beginning of this component if `disableExpiryMessagingForLearnerCredit && isBudgetExpired` ?

Reply:
Yep good catch. Its better to consolidate redundant logic 👍🏽